cli/nailgun-server/src/main/scala/com/rackspace/com/papi/components/checker/cli/NailgunServer.scala

```diff
@@ -1,116 +1,95 @@
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
 package com.rackspace.com.papi.components.checker.cli
 
-import java.net.InetAddress
-
-import java.util.Date
-import java.util.Timer
-import java.util.TimerTask
-
-import java.io.InputStream
-import java.io.PrintStream
 import java.io.IOException
-
-import java.nio.file.FileSystems
-import java.nio.file.Paths
-import java.nio.file.Path
-import java.nio.file.Files
-import java.nio.file.FileVisitResult
+import java.net.InetAddress
 import java.nio.file.FileVisitResult._
-import java.nio.file.SimpleFileVisitor
-import java.nio.file.WatchService
+import java.nio.file._
 import java.nio.file.StandardWatchEventKinds._
-
 import java.nio.file.attribute.BasicFileAttributes
-
-import javax.xml.datatype.DatatypeFactory
-import javax.xml.datatype.Duration
-
-import org.clapper.argot.ArgotConverters._
-import org.clapper.argot.{ArgotParser, ArgotUsageException}
-
-import com.rackspace.com.papi.components.checker.wadl.WADLCheckerBuilder
+import java.util.{Date, Timer, TimerTask}
 
 import com.martiansoftware.nailgun.NGServer
+import javax.xml.datatype.DatatypeFactory
+import scopt.OptionParser
 
 object NailgunServer {
   //
   //  Defaults
   //
-  val HOST_DEFAULT : String = "127.0.0.1"
-  val PORT_DEFAULT : Int = 2113
-  val DURATION_DEFAULT : String = "PT2H"
-  val DONT_EXIT_ON_SHUTDOWN  : Boolean = false
+  val HOST_DEFAULT: String = "127.0.0.1"
+  val PORT_DEFAULT: Int = 2113
+  val DURATION_DEFAULT: String = "PT2H"
+  val DONT_EXIT_ON_SHUTDOWN: Boolean = false
 
   val title = getClass.getPackage.getImplementationTitle
-  val version = getClass.getPackage.getImplementationVersion
+  val implVersion = getClass.getPackage.getImplementationVersion
 
-  def parseArgs (args : Array[String]) : Option[(String, Int, Long, Boolean)] = {
+  def parseArgs(args: Array[String]): Option[(String, Int, Long, Boolean)] = {
 
-    val parser = new ArgotParser("checker-nailgun-server", preUsage=Some(s"$title v$version"))
+    var duration = DURATION_DEFAULT
+    var port = PORT_DEFAULT
+    var host = HOST_DEFAULT
+    var dontExitOnShutdown = DONT_EXIT_ON_SHUTDOWN
 
-    val duration = parser.option[String] (List("d", "duration"), "duration",
-                                          s"The duration of time (in ISO 8601 format) before the server shuts down. Default: $DURATION_DEFAULT")
+    val parser = new OptionParser[Unit]("checker-nailgun-server") {
+      head(s"$title v$implVersion")
 
-    val port = parser.option[Int] (List("p", "port"), "port",
-                                   s"The port for the server to listen to. Default: $PORT_DEFAULT")
+      help('h', "help").text("Display usage.")
 
-    val host = parser.option[String] (List("H", "host"), "host",
-                                      s"The host address for the server to listen to. Default: $HOST_DEFAULT")
+      version("version").text("Display version.")
```

Comment:
I think `help` and `version` both show up in the usage text in the same relative position as they are declared. Since `CheckerParser` puts them at the bottom, do we want to do the same here for consistency?

Reply:
I actually just moved them to the top, because of the joys of static initialization they were probably mid list before.